ls1intum/Artemis
# [Fix] Fix results not showing in participation-submission component

### Checklist

### Motivation and Context



The problem was that the results and assessor were not shown


### Description

- eagerly load the assessor 
- fix html to support multiple results


### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Click on Exercises -> Participations 
4. Click on Submission Id
5. Make sure Assessor and Assessments / Results are correctly shown 

```
            .-~~\
           /     \ _
           ~x   .-~_)_
             ~x".-~   ~-.
         _   ( /         \   _
         ||   T  o  o     Y  ||
       ==:l   l   <       !  I;==
          \\   \  .__/   /  //
           \\ ,r"-,___.-'r.//
            }^ \.( )   _.'//.
           /    }~Xi--~  //  \
          Y    Y I\ \    "    Y
          |    | |o\ \        |
          |    l_l  Y T       |  -Row
          l      "o l_j       !
           \                 /
    ___,.---^.     o       .^---.._____
"~~~          "           ~            ~~~"
```
